content/en/load_balancer.md

```diff
@@ -3,12 +3,15 @@ title: Load Balancer
 status: Feedback Appreciated
```

Comment:
When this PR is okay to be merged, please change the `status` to `Completed`:

```suggestion
status: Completed
```

Reply:
Does this term need to be changed by me or by the maintainers?